.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,7 @@ Ensure you completed **all of the steps** below before submitting your pull requ
 
 - [ ] Added natspec comments?
 - [ ] Ran `forge snapshot`?
+- [ ] Ran `pnpm gas:report`?
```

Comment:
The `pnpm` command is called `gas-report`.
So either we change this here to `gas-report`, or we change the command name to `gas:report`.

I think both are fine. Feel free to choose